docs/application/dotnet/tutorials/create-dotnet-hybrid-package.md

```diff
@@ -39,37 +40,39 @@ $[TIZEN_STUDIO_DIR]/tools/ide/bin$ ./tizen package -t wgt -s [CERTIFICATE_PROFIL
 
     ![Create Web Widget 4](./media/create_web_sample_cs_4.png)
 
-2. Check manifest file of web application
+2. Check the **config.xml** file of the Web application:
 
     ```xml
     <?xml version="1.0" encoding="UTF-8"?>
     <widget xmlns="http://www.w3.org/ns/widgets" xmlns:tizen="http://tizen.org/ns/widgets" id="http://yourdomain/WorldClockWidget" version="1.0.0" viewmodes="maximized">
         <access origin="*" subdomains="true"></access>
-        <tizen:app-widget id="TestHybrid.WorldClockWidget.WorldClockWidget" primary="true">
+        <tizen:app-widget id="NtKvDqWLe7.WorldClockWidget.WorldClockWidget" primary="true">
             <tizen:widget-label>World Clock Widget</tizen:widget-label>
-            <tizen:widget-content src="widget/Widget/index.html">
-                <tizen:widget-size preview="widget/Widget/preview.png">2x2</tizen:widget-size>
+            <tizen:widget-content src="widget/WorldClockWidget/index.html">
+                <tizen:widget-size preview="widget/WorldClockWidget/preview.png">2x2</tizen:widget-size>
             </tizen:widget-content>
         </tizen:app-widget>
-        <tizen:application id="TestHybrid.WorldClockWidget" package="TestHybrid" required_version="2.3.2"/>
+        <tizen:application id="NtKvDqWLe7.WorldClockWidget" package="TestHybrid" required_version="2.3.2"/>
         <content src="index.html"/>
         <feature name="http://tizen.org/feature/screen.size.all"/>
-        <icon src="icon.png"/>
-        <name>WorldClockWidget</name>
-        <tizen:privilege name="http://tizen.org/privilege/application.launch"/>
-        <tizen:profile name="wearable"/>
+		<icon src="icon.png"/>
+		<name>WorldClockWidget</name>
+		<tizen:privilege name="http://tizen.org/privilege/application.launch"/>
+		<tizen:profile name="wearable"/>
     </widget>
     ```
-    package id of web application is set like below :
-     __*package="TestHybrid"*__
+    > [!NOTE]
+	> Ensure that the package ID of the Web application is set to **TestHybrid**. For example, `package="TestHybrid"`.
```

Comment:
Suggestion. Isn't it better not to use "For example" here, as `package="TestHybrid"` is not an example but an actual string that should be in `package="TestHybrid"` form?

Same issue to line 95.

Reply:
@mijin85cho,

The reason to use"For example" here is to explain the sentence in the note. How the package ID should be.

Reply:
@surbhig12 I think this line is the additional description for line 55, the part of the sample code above.

![image](https://user-images.githubusercontent.com/9958374/84842463-274ece80-b080-11ea-9639-34f9d6de51b9.png)

As I don't think this line as an example explanation but the description to the above sample code, "For example" sounds weird to me. Why do we suddenly give example to the user while that code is already provided above?

Reply:
@mijin85cho,
I understand your point. I will remove "For example.."

Reply:
Fixed it. Thanks.